lib/reporters/verbose.js

```diff
@@ -6,13 +6,16 @@ var plur = require('plur');
 var repeating = require('repeating');
 var colors = require('../colors');
 
-Object.keys(colors).forEach(function (key) {
-	colors[key].enabled = true;
-});
-
-function VerboseReporter() {
+function VerboseReporter(options) {
 	if (!(this instanceof VerboseReporter)) {
-		return new VerboseReporter();
+		return new VerboseReporter(options);
+	}
+
+	this.options = Object.assign({}, options);
```

Comment:
Assigning `this.options` seems unnecessary since the constructor didn't receive `options` before. Nothing else is using `this.options`.